src/cdk/testing/tests/testbed.spec.ts

```diff
@@ -1,5 +1,5 @@
-import {HarnessLoader} from '@angular/cdk-experimental/testing';
-import {TestbedHarnessEnvironment} from '@angular/cdk-experimental/testing/testbed';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
```

Comment:
I'm slightly worried about these imports with our current gulp setup as it only supports a primary entry-point and multiple secondary entry-points for a package. This seems like a tertiary entry-point that might not work with Gulp (once the CDK is packaged)

I'd propose that we still move on with this and figure that out in follow-ups.

Reply:
Yeah, I was hoping we would be building our release with Bazel at this point, but since we're not we'll either have to do some work to switch to Bazel or add tertiary EPs to our gulp scripts

Reply:
Yeah, we'll see if we can make the switch. If everything goes well, it should be with bazel.